zeppelin-zengine/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -428,6 +428,7 @@ public String getConfDir() {
     // Decide when new note is created, interpreter settings will be binded automatically or not.
     ZEPPELIN_NOTEBOOK_AUTO_INTERPRETER_BINDING("zeppelin.notebook.autoInterpreterBinding", true),
     ZEPPELIN_CONF_DIR("zeppelin.conf.dir", "conf"),
+    ZEPPELIN_INTERPRETER_REPO_DIR("zeppelin.user.interpreter.dir", "interpreter"),
```

Comment:
@corneadoug because Interpreter is loaded by `conf/interpreter.sh` and other pluggable component are not, copying dependencies in a directory is only requirement of `conf/interpreter.sh`. So this configuration shouldn't be generalized for other pluggable module.

Dependencies for other pluggable components will directly loaded from local repository for dependency loader, which is configured by ZEPPELIN_DEP_LOCALREPO.